### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dev7ch**
try to make markup working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ To connect your hosting server with your repo is some configuration needed.
 
 Add something like this to your configuration file:
 
-```shell
+```php
```
Comment (nadar) [inline README.md]: @dev7ch does not look like php commands...

File: README.md [hunk 2]
```diff
@@ -148,21 +148,21 @@ Host
 
 make sure the that the correct permissions and owner are applied to the created config file:
 
-```shell
+```php
```
Comment (nadar) [inline README.md]: same

File: README.md [hunk 3]
```diff
@@ -148,21 +148,21 @@ Host
...
 chmod 600 ~/.ssh/config
 chown $USER ~/.ssh/config
 ```
 
 5. Adding BitBucket to `known_hosts` on your server.
 
-```shell
+```php
```
Comment (nadar) [inline README.md]: same

File: README.md [hunk 4]
```diff
@@ -148,21 +148,21 @@ Host
...
 vim ~/.ssh/known_hosts
 ```
 
-Usually the RSA fingerpint is etablisch automatically on approval if you run `git clone` via ssh, but here we need to add it manually.
+Usually the RSA fingerpint is etablished automatically on approval if you run `git clone` via ssh, but here we need to add it manually.
```
Comment (nadar) [inline README.md]: established

File: README.md (hunk 1 above)
Comment (dev7ch) [reply README.md]: https://luya.io/guide/module/luyadev---luya-deployer#hosts-and-deployment-key

Comment (dev7ch) [reply README.md]: doenst work with ```shell

Comment (nadar) [reply README.md]: but its not php. use `sh` instead

**Pull request merged**